pom.xml

```diff
@@ -40,7 +40,7 @@
             <dependency>
                 <groupId>com.backbase.dbs</groupId>
                 <artifactId>banking-services-bom</artifactId>
-                <version>2.17.3</version>
+                <version>2.18.0-cr.37</version>
```

Comment:
you only can use versions that have been released to public repo

Reply:
Hi Richard, the teams need to use this particular version in order of them to ingest data with the current in development version (soon to be released) 2.18.0 dbs version to validate their changes.

Reply:
That's only possible with a release that has been built internally. I have added this local build as `pre-release` (https://github.com/Backbase/bb-fuel/releases/tag/bb-fuel-2.5.14-alpha) that can be used in the pipelines by enabling `PRERELASE` and latest version

Reply:
Oh, okay, thanks for letting me know, switched back to 2.17.3